package.json

```diff
@@ -35,7 +35,8 @@
 		"@ui5/fs": "^2.0.1",
 		"@ui5/logger": "^2.0.0",
 		"@ui5/project": "^2.1.5",
-		"@ui5/server": "^2.2.1"
+		"@ui5/server": "^2.2.1",
+		"handlebars": "^4.7.6"
```

Comment:
`handlebars` should be a devDependency, similarly to jsdoc. It's a bit weird in this repository since it doesn't contain any real code (yet). But in other repositories this would be a devDependency since it is only required for building the documentation (hence during development).